ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -586,3 +579,36 @@ export const RuntimeConfigurationPanel = fp.flow(
     );
   }
 );
+
+export interface RuntimeConfigurationPanelProps {
+  onClose?: () => void;
+  initialPanelContent?: PanelContent;
+  creatorInitialCreditsRemaining?: number;
+  profileState: ProfileStore;
+}
+
+export const RuntimeConfigurationPanel = ({
+  profileState,
+  onClose = () => {},
+  initialPanelContent,
+  creatorInitialCreditsRemaining,
+}: RuntimeConfigurationPanelProps) => {
+  const workspace = useCurrentWorkspace();
+  const { namespace } = workspace;
+  const { isLoaded } = useRuntimeAndDiskStores(namespace);
```

Comment:
I would not guess that this line *initializes* the stores, so a comment to that effect would be helpful.  Unfortunate that hooks need to start with "use" in this case.